docs/en/sql-reference/statements/insert-into.md

```diff
@@ -91,6 +91,13 @@ INSERT INTO t FORMAT TabSeparated
 
 You can insert data separately from the query by using the command-line client or the HTTP interface. For more information, see the section “[Interfaces](../../interfaces)”.
 
+:::note
+If you want to specify `SETTINGS` for `INSERT` query then you have to do it _before_ `FORMAT` clause since everything after `FORMAT format_name` is treated as data. For example:
```

Comment:
```suggestion
If you want to specify `SETTINGS` for an `INSERT` query then you have to do it _before_ the `FORMAT` clause, since everything after `FORMAT format_name` is treated as data. For example:
```